packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1924,6 +1921,22 @@ class StackViewport extends Viewport implements IStackViewport {
     });
   }
 
+  /**
+   * Given the cornerstone image object it will render it in the viewport
+   * This method is used by some utilities, for loading an actual image use
+   * the setStack method and not this one.
```

Comment:
Could you add a comment about where to look for the rendered content?  It feels like this should be able to render to a canvas or the like, but it isn't obvious that one can't do that.